cfy_manager/components/restservice/restservice.py

```diff
@@ -298,7 +293,8 @@ def _configure():
 
 def install():
     logger.notice('Installing Rest Service...')
-    _install()
+    source_url = config[RESTSERVICE][SOURCES]['restservice_source_url']
+    yum_install(source_url)
```

Comment:
If this is the only used of the source URL, maybe we can cram into a single line?